db/migrate/20150513121721_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
Migrations should serve only for schema updates and not data migrations. They are basically irreversible and I already had to clear this issue in one project where migrations were not running from start to end due to renamed models, columns, missing data.

The proper way is actually to create a custom rake task under `/lib/tasks`. Read more about this [here](http://guides.rubyonrails.org/command_line.html#custom-rake-tasks).